python/py-autobahn/Portfile

```diff
@@ -16,16 +16,23 @@ homepage            https://crossbar.io/autobahn
 master_sites        pypi:a/autobahn
 distname            autobahn-${version}
 
-checksums           rmd160  90a33d2f405a0581681cb0be4354507250d46202 \
-                    sha256  8fb9f3e7f6de5b7ee5479a56582873b927cc25d72319d0a86f43d992048d8171 \
-                    size    682775
+checksums           rmd160  d38811290f505c10cc941abd4cbd745e15f3136d \
+                    sha256  86bbce30cdd407137c57670993a8f9bfdfe3f8e994b889181d85e844d5aa8dfb \
+                    size    1260579
 
-python.versions     27 37 38
+python.versions     38
```

Comment:
It looks like the only port currently depending on py-autobahn is py-buildbot, only available as py38, so that's fine... but there is a comment in py-magpy about the desire to use py-autobahn, and py-magpy is only available as py27, and there is a comment saying that py3 versions are broken. If that comment is still accurate then keeping py27-autobahn may be desired.

Reply:
Don't worry about magpy.

First of all, they released a new version that requires Python >= 3.6 (they say it's in theory still somewhat compatible with 2.7, but extra effort is needed to get it working). The only problem is that Python 3 means wxPhoenix which I still didn't come around at properly packaging (it's a lot more annoying to do it nicely compared to wxPython).

Second, I believe that they got tired of every version of autobahn being incompatible with the previous one (that's what they've been discussing a few years ago), and most likely they dropped the dependency. Or at least I no longer see anything in https://github.com/geomagpy/magpy/blob/master/setup.py. (Autobahn was used in some optional modules of magpy. I don't know what's the current state as I haven't touched the software for quite a while.)

Third, it's a pretty niche piece of software and I wouldn't be surprised if there are only three labs in the world actually using it.

If buildbot master works with this, that's all that matters.

Reply:
I've have buildbot master 2.8.2 running with python38 and whatever versions of the dependencies were current as of a week or so ago and it's working fine. But I could only find references to autobahn in buildbot's documentation about the optional "wamp" message queue implementation, which I'm not using yet but which I want to try using, but which also requires [crossbar](https://trac.macports.org/ticket/60974) which Herby is in the process of trying to add for me, hence all his recent python module submissions and updates.